src/data/dialogmote/dialogmoteTexts.ts

```diff
@@ -1,21 +1,57 @@
+import { Malform } from "../../context/malform/MalformContext";
+
 export const innkallingTexts = {
   arbeidstaker: {
```

Comment:
Vil det komme nynorsk-tekster til arbeidsgiver og behandler senere eller er det bare arbeidstakeren som kan få nynorsk-brev?

Reply:
Ja, kommer senere! Men vi ville bare lage en draft så dere kan se mønsteret før vi endrer all teksten 🙃

Reply:
Skjønner! 
Lurte litt på om det kunne vært nice å ha to innkallingTexts-objekter her, et for bokmål og et for nynorsk (som har de samme nøklene, men ulike verdier/tekster). Og så en funksjon som tok malform som parameter og ga deg riktig objekt med alle tekstene som kan brukes fra document-hooken. (Litt ala sånn man henter ut tekstene til forhandsvarsel-dokumentet).
Da slipper man å hente ut hver enkelt tekst med malform som parameter i document-hooken.

Reply:
Kan vise noe kode hvis det trengs :slightly_smiling_face:

Reply:
Yes, vi skal prøve oss litt frem rundt dette nå 👍🏼